Cargo.toml

```diff
@@ -1,2 +1,6 @@
 [workspace]
 members = ["spotify_player", "lyric_finder"]
+resolver = "2"
+
+[profile.release]
+debug = 1
```

Comment:
@aome510 Would this reduce performance to leave this in for the release build?

Reply:
In theory, yes, but I believe the effect should be negligible. Enable debug symbol can help provide better backtrace and make profiling easier.